docs/vsphere_ci/vcenter_copy_template/sourcevcs8e

```diff
@@ -0,0 +1,6 @@
+export GOVC_URL=https://vcs8e-vc.ocp2.dev.cluster.com
+export GOVC_USERNAME=xxxxxxx@vsphere.local
+export GOVC_PASSWORD='xxxxxxx'
+export GOVC_INSECURE=true
+export GOVC_DATACENTER=IBMCloud
+export GOVC_RESOURCE_POOL='QE Resource Pool'
```

Comment:
these templates seem very similar to each other
Is there a way to combine them?

Reply:
not really, they are example configs for different vcenter envs which each may have different creds